### PR TITLE
Parser - call command

### PR DESCRIPTION
**Pull request by osvegn**
Pull request link to this issue: #31 .

### CODE REVIEW

File: server/src/parser/call_command.c [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+#include <stddef.h>
+#include <stdbool.h>
+
+int call_command(char **arguments)
+{
+    cli_command_t *commands = get_cli_commands();
+    void *retval = NULL;
+    bool command_found = false;
+
+    for (int i = 0; i < 14; i++) {
```
Comment (LucasTesnier) [inline server/src/parser/call_command.c:20]: Add a macro for Command_Tab size

File: server/src/parser/cli_commands.c [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+** cli_commands
+*/
+
+/// \file server/src/parser/cli_commands.c
+
+#include "cli_commands.h"
+#include <stddef.h>
+
+/// \brief List all available command
+static const cli_command_t commands[14] = {
```
Comment (LucasTesnier) [inline server/src/parser/cli_commands.c:14]: Add a macro for Command_Tab size

**Review by LucasTesnier — CHANGES_REQUESTED** [review]
NTR!

File: server/src/parser/cli_commands.c [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    {"/use", {"", NULL}, &use_command},
+    {"/create", {"", "", NULL}, &create_command},
+    {"/list", NULL, &list_command},
+    {"/info", NULL, &info_command}
+};
+
+const cli_command_t *get_cli_commands()
+{
+    return (commands);
+}
```
Comment (Cavonstavant) [inline server/src/parser/cli_commands.c:34]: Consider adding a function to retrieve one command info by giving it's name as a parameter

**Review by Cavonstavant — APPROVED** [review]
LGTM

**Review by LucasTesnier — APPROVED** [review]
NTR!

**Pull request merged**